src/plugins/vis_type_vislib/public/vislib/components/tooltip/_pointseries_tooltip_formatter.js

```diff
@@ -55,7 +56,15 @@ export function pointSeriesTooltipFormatter() {
 
     if (datum.y !== null && datum.y !== undefined) {
       const value = datum.yScale ? datum.yScale * datum.y : datum.y;
-      addDetail(currentSeries.label, currentSeries.yAxisFormatter(value));
+      let label = currentSeries.label;
+
+      // For stacked charts the y axis data is only available in the raw table
+      const tableColumns = get(datum, 'yRaw.table.columns');
+      if (tableColumns && tableColumns.length > 2) {
+        const yColumn = datum.yRaw && datum.yRaw.column ? tableColumns[datum.yRaw.column] : {};
```

Comment:
tableColumns is get from `get(datum, 'yRaw.table.columns')`, so the if clause `if(tableColumns && ...)` already test the existence of `datum.yRaw.table.columns`.

`datum.yRaw.table.columns` exists then `datum.yRaw` should exist right?
so can `yColumn` just be `yColumn = datum.yRaw.column ? tableColumns[datum.yRaw.column] : {};`

Reply:
Good callout. I didn't notice the duplicate check. I think i was working as if this was a typescript file which would definitely complain if i ignored that check. Will remove it.